scripts/edit_command_line.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+if [ -z "$1" ]; then
+	printf 'missing input file\n'
+	exit 2
+fi
+
+# set input_file to the last argument
+# the last argument is used because in the case of zsh, commands maybe be passed before the filename e.g., (-c)
+for input_file; do true; done
+
+# after exiting this script and before it has been read by kitty-scrollback.nvim
+# the contents of the original input_file may be altered
+# avoid this by copying the input_file to a new file that will be referenced
+ksb_input_dir=$(mktemp -d)
+ksb_input_file="$ksb_input_dir/input.ksb_editcommand"
+cp "$input_file" "$ksb_input_file"
+
+kitty @ kitten /Users/mike/gitrepos/kitty-scrollback.nvim/python/kitty_scrollback_nvim.py \
```

Comment:
Hardcoded your path here

Reply:
It works after patching it manually though

Reply:
thanks for pointing this out! I just removed the hardcoded path. Could you try with latest?

Reply:
works :+1: